pkg/node/address.go

```diff
@@ -566,6 +570,30 @@ func SetK8sNodeIP(ip net.IP) {
 	k8sNodeIP = ip
 }
 
+func SetWireguardIPv4(ip net.IP) {
+	wireguardIPv4 = ip
+}
```

Comment:
Shouldn't we ensure that `ip` is actually v4 and not v6?

Reply:
Unless you feel strongly about this, I'll leave this as is, especially since all the other setters in this package are also very bare bones, c.f. `SetIPv4`, `SetIPv4Loopback`, `SetIPv4AllocRange`

Panicking is probably the right choice here and returning an error would also not help much, as the existing invocation site already necessarily has to check for IPv4.